app/presenters/search_result_presenter.rb

```diff
@@ -8,29 +8,80 @@ class SearchResultPresenter
            :es_score,
            to: :search_result
 
-  def initialize(search_result, metadata)
+  def initialize(search_result, metadata, doc_index, doc_count, finder_name, debug_score, highlight)
     @search_result = search_result
     @metadata = metadata
+    @index = doc_index + 1
+    @count = doc_count
+    @finder_name = finder_name
+    @debug_score = debug_score
+    @highlight = highlight
   end
 
   def to_hash
     {
-      title: title,
-      link: link,
-      summary: summary,
-      is_historic: is_historic,
-      government_name: government_name,
-      metadata: metadata,
-      show_metadata: show_metadata,
-      format: format,
-      es_score: es_score,
+      link: {
+        text: title,
+        path: link,
+        description: summary_text,
+        data_attributes: {
+          track_category: "navFinderLinkClicked",
+          track_action: "#{@finder_name}.#{@index}",
+          track_label: link,
+          track_options: {
+            dimension28: @count,
+            dimension29: title
+          }
+        }
+      },
+      metadata: structure_metadata,
+      metadata_raw: metadata,
+      subtext: subtext,
+      highlight: @highlight,
```

Comment:
Using a private `attr_reader :highlight` is preferred over using a variable `@highlight`. See `attr_reader :search_result` below.

Reply:
I never really get why that is.  ```@highlight``` makes me realise this is an instance variable which attr_reader does not.

Reply:
I do this because Sandy Metz tells me to 😄. The main reason here is it's consistent in this codebase.

Reply:
Hmm. Let me know who wins this discussion and I'll go with the victor's preference.

Reply:
Consistency makes sense here yes. Sandi is almost always right, but in this case wrong 😉